src/test/resources/features/get_excel_report.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Generate and Retrieve Excel Financial Report
+
```

Comment:
Looks great. In the future we could test if the report ID is missing and if the user is not authenticated